### DISCUSSION TITLE
[2024-11-08] AMER Building DevRel Foundation Community Call

### DISCUSSION THREAD

**Discussion by katiewaz1977** [DevRel-Foundation/governance#117] [Meeting Notes]
> Part of our community calls are being used as Open Office Hours to reach community consensus on Foundation Next Steps. If you would like to get involved, this is a safe and welcoming space to start!

> Whether you are new to the call or a regular participant, you can add your discussion topics as a comment below or bring them up during the call. You can also share DevRel use cases, best practices, frameworks, and more by opening a new issue in the [GitHub Discussions](https://github.com/DevRel-Foundation/governance/blob/main/working-groups/themes.md)), which serves as the Foundation’s public forum. Additionally, we are always looking for 5-10 minute presentations in future community calls to spark discussion with the rest of the attendees and async.

## Agenda

👋 Intros  - If we have a large group, we'll use chat
🧹 Housekeeping
- Code of conduct ([link](https://github.com/DevRel-Foundation/governance/blob/main/code_of_conduct.md)). 
Instances of abusive, harassing, or otherwise unacceptable behavior, please report at conduct@dev-rel.org
- Antitrust Policies ([link](https://www.linuxfoundation.org/legal/antitrust-policy)). At meetings among actual or potential competitors, there is a risk that participants in those meetings may improperly disclose or discuss information in violation of the Antitrust Laws, or otherwise act in an anti-competitive manner. To avoid this risk, DRF participants must adhere to[ the policies available in conduct of meeting section](https://www.linuxfoundation.org/legal/antitrust-policy) when participating in DRF calls. To contact LF regarding matters addressed by this Antitrust Policy, please send an email to legal@linuxfoundation.org, and reference “Antitrust Policy” in the subject line.
- Intro to roles: facilitator, moderator, scribe

🤔 What we're doing in this call

The Developer Relations Foundation mission:
_The mission of the Developer Relations Foundation project is to elevate the professional practice of developer relations and increase awareness of it as a driver of business value_

### Open Office Hours

Time to listen to the voices of community members, answer questions, and tackle blockers together. Topics can be related to:

- Solve questions about working groups based on the [phased timeline](https://github.com/orgs/DevRel-Foundation/projects/7/views/1)
- Open discussion on the challenges in LATAM countries, specifically in Brazil on the DevRel Value - How can the foundation support the growth of this initiative so that Latin America can recognize the strategic importance of roles like DevRel, staff engineers, and specialists — before it’s too late? (raised by @thiago4int )

- Hey YOU! 🙋🏽‍♀️🙋🏽🙋🏻🙋🏼‍♀️🙋🏻 Whether you are new to the call or a regular participant, the community shapes the conversations happening in DevRel Foundation calls! Please add your discussion topics as a comment below or bring them up during the call. You can also share any DevRel use cases, best practices, frameworks, etc as a forum, by opening a new issue in the GH Discussions.

## Essential Information

* 🕛 Friday November 8, 12:00 - 1:00 pm EST
* 📝 [g.doc agenda](https://docs.google.com/document/d/1I5dX4RbvJyt4Z2dTWsS1BjmzRQV8RHOJ5klJ_tZa_eg/edit?usp=sharing)

| Frequency | Duration | Zoom link | Join Discord |
| -- | -- | -- | -- |
| bi-weekly | 1 hour | [Here](https://zoom.us/j/97083888502?pwd=5WQDWGN7U08Nvw6r12nQB48Kc5hqg2.1) | [Join](https://discord.gg/2jZ833bx) |

**Comment by thiago4int** [comment]
Hello, I’m Thiago, and I’d like to address a pressing challenge we face in Brazil and Latin America. Currently, there’s little to no perceived value for the DevRel (Developer Relations) role in our region. It took two years of sustained effort for the FinOps Foundation to gain traction, and similarly, there’s a need for broader education on DevRel here, including its clear definitions and real benefits, such as talent retention and increased business value.

Our companies are losing talented professionals because certain roles, like DevRel, are entirely overlooked. As a result, developers are often pushed into tech lead roles without adequate support, and architects end up working in pre-sales. How can the foundation support the growth of this initiative so that Latin America can recognize the strategic importance of roles like DevRel, staff engineers, and specialists—before it’s too late?

**Reply by anajsana** [reply]
Thank you! we will add it to the agenda right away!
